src/Tribe/Admin/Manager/Service_Provider.php

```diff
@@ -56,7 +56,7 @@ public function ajax_handle_admin_manager() {
 		}
 
 		// Get the request vars.
-		$vars = tribe_get_request_vars();
+		$vars = $_REQUEST;
```

Comment:
Maybe leave a note to future self / colleagues on why we need to use the `$_REQUEST` here?

Reply:
Added